apps/codeforafrica/next.config.js

```diff
@@ -12,6 +12,10 @@ const outputFileTracingRoot = PROJECT_ROOT
   ? path.resolve(__dirname, PROJECT_ROOT)
   : undefined;
 
+const ghostUrl =
+  process.env.GHOST_ADMIN_URL?.trim() || process.env.GHOST_URL?.trim();
```

Comment:
@kilemensi  do we have a case where we would need `GHOST_ADMIN_URL` and `GHOST_URL`? Or why can't we always use just `GHOST_URL`?

Reply:
This directly comes from Ghost @esirK : Ghost seems to allow admin dashboard and site to use completely different protocols and domains, more in [docs](https://ghost.org/docs/config/#running-ghost-with-config-env-variables).